src/modules/mavlink/mavlink_main.h

```diff
@@ -334,6 +335,8 @@ class Mavlink : public device::VDev
 
 	int 			get_socket_fd () { return _socket_fd; };
 
+	struct sockaddr_in * get_socket_source_address() {return &_src_addr;};
```

Comment:
Travis CI delivers a concise build log. If you had already the hunch, why not push a commit that ifdefs this out?